calicolipidlibrary/MIP2C.py

```diff
@@ -45,39 +45,23 @@ def theoreticalDigest(self):
 			return(FRAGMENTS)
 
 
-		@classmethod
-		def generateLibrary(cls,target=None, mode="pos"):
+
+		def generateLibrary(self, target=None, mode="pos"):
```

Comment:
This is the first definition of generateLibrary().  It has a customized `chains` variable